outgoingra/src/main/java/no/bankid/outgoing/ra/SendActivationCodeRequestBody.java

```diff
@@ -0,0 +1,23 @@
+package no.bankid.outgoing.ra;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.UUID;
+
+@Schema(description = "Request body content for asking an RA to send an activation code to an endUser")
+public class SendActivationCodeRequestBody extends AuthenticationBody {
```

Comment:
Rename VerificationCode... to be inline with api name

Reply:
removed